src/imars3d/__init__.py

```diff
@@ -1,173 +1,4 @@
-    normalization(ct_series, dfs, obs, normalized_ct)
-    return normalized_ct
-
-
-@dec.timeit
-def correct_tilt(ct_series, tilt=None, workdir="work", max_npairs=10, parallel=True):
-    if tilt is None:
-        tiltcalc = components.TiltCalculation(workdir=workdir, max_npairs=max_npairs)
-        tilt = tiltcalc(ct_series)
-
-    # only correct if the tilt is large
-    if abs(tilt) > 0.002:
-        tiltcorrected_series = io.ImageFileSeries(
-            os.path.join(workdir, "tiltcorrected_%07.3f.tiff"),
-            identifiers=ct_series.identifiers,
-            name="Tilt corrected CT",
-            mode="w",
-        )
-        tiltcorr = components.TiltCorrection(tilt=tilt)
-        tiltcorr(ct_series, tiltcorrected_series, parallel=parallel)
-        return tiltcorrected_series, tilt
-    return ct_series, tilt
-
-
-@dec.timeit
-def correct_intensity_fluctuation(ct_series, workdir="work"):
-    intfluct_corrected_series = io.ImageFileSeries(
-        os.path.join(workdir, "intfluctcorrected_%07.3f.tiff"),
-        identifiers=ct_series.identifiers,
-        name="Intensity fluctuation corrected CT",
-        mode="w",
-    )
-    ifcorr = components.IntensityFluctuationCorrection()
-    ifcorr(ct_series, intfluct_corrected_series)
-    return intfluct_corrected_series
-
-
-@dec.timeit
-def build_sinograms(ct_series, workdir="work", parallel=True, parallel_nodes=None):
-    sinograms = io.ImageFileSeries(
-        os.path.join(workdir, "sinogram_%i.tiff"),
-        name="Sinogram",
-        mode="w",
-    )
-    if parallel:
-        proj = components.Projection_MP(num_workers=parallel_nodes)
-    else:
-        proj = components.Projection()
-    proj(ct_series, sinograms)
-    return ct_series.identifiers, sinograms
-
-
-def ring_artifact_removal_Ketcham(sinograms, workdir="work", parallel=True, filename_template=None, **kwds):
-    if filename_template is None:
-        filename_template = "rar_sino_%i.tiff"
-    rar_sino = io.ImageFileSeries(
-        os.path.join(workdir, filename_template), identifiers=sinograms.identifiers, name="RAR_Sinogram", mode="w"
-    )
-    filter = components.RingArtifactRemoval_Kectham(**kwds)
-    filter(sinograms, rar_sino, parallel=parallel)
-    return rar_sino
-
-
-def reconstruct(angles, sinograms, workdir="work", filename_template=None, **kwds):
-    """reconstruct
-
-    * angles: ct scan angles in degrees
-    """
-    filename_template = filename_template or "reconstructed_%i.tiff"
-    recon_series = io.ImageFileSeries(
-        os.path.join(workdir, filename_template),
-        identifiers=sinograms.identifiers,
-        name="Reconstructed",
-        mode="w",
-    )
-    import numpy as np
-
-    theta = angles * np.pi / 180.0
-    from imars3d.recon.mpi import recon
-
-    recon(sinograms, theta, recon_series, **kwds)
-    return recon_series
+try:
+    from ._version import __version__  # noqa: F401
+except ImportError:
+    __version__ = "unknown"
```

Comment:
To get pass the code coverage error, we should either exclude this file or add a test to check the version number.